formats/protobuf/commonMain/src/kotlinx/serialization/protobuf/ProtoId.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2017-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization.protobuf
+
+import kotlinx.serialization.*
+
+@SerialInfo
+@Target(AnnotationTarget.PROPERTY)
+@Deprecated(message = "SerialId is renamed to ProtoId to better reflect its semantics",
+    level = DeprecationLevel.WARNING, replaceWith = ReplaceWith("ProtoId(id)")
+)
+annotation class SerialId(val id: Int)
+
+@SerialInfo
+@Target(AnnotationTarget.PROPERTY)
+annotation class ProtoId(val id: Int)
```

Comment:
You also can add documentation here

Reply:
No, it's not my goal. Especially taking into account the fact that I am not really familiar with out protobuf implementation